sqlglot/dialects/databricks.py

```diff
@@ -23,6 +25,21 @@ class Parser(Spark.Parser):
             TokenType.COLON: exp.JSONExtract,
         }
 
+        def _parse_generated(self, kind: t.Optional[str] = None) -> exp.ComputedColumnConstraint:
+            self._match_text_seq("ALWAYS")
+            self._match(TokenType.ALIAS)
+            this = self._parse_expression()
+
+            return self.expression(
+                exp.ComputedColumnConstraint,
+                this=this,
+            )
+
+        CONSTRAINT_PARSERS = {
```

Comment:
move constraint parsers above the function

Reply:
Ditto - if we override `_parse_generated_as_identity` we can remove this altogether.